blockedModeSetup.js

```diff
@@ -0,0 +1,54 @@
+document.addEventListener('DOMContentLoaded', function() {
+	let textField = document.getElementById('duration');
+	textField.focus();
+	textField.select();
+	
+	document.getElementById('duration').addEventListener('keyup', function(event) {
+		event.preventDefault();
+		if (event.keyCode === 13) {
+			document.getElementById("start").click();
```

Comment:
You could put a code exectued by the start button listener to a separated function and use that function in enter press listener.